src/bytes/src/bytes.cairo

```diff
@@ -31,12 +32,12 @@ const BYTES_PER_ELEMENT: usize = 16;
 ///  - size: the number of bytes in the Bytes
 ///  - data: the data of the Bytes
 #[derive(Drop, Clone, PartialEq, Serde)]
-struct Bytes {
+pub struct Bytes {
     size: usize,
-    data: Array<u128>
+    pub data: Array<u128> // TODO should this be pub?
```

Comment:
no, add a getter or smth

Reply:
Added a data getter and impl the index view to solve some moved value issue.
Quite an improvement imho :) 
Please, double check "compare_bytes()" I had to remove a check.